tests/support/activemq/topic_details.py

```diff
@@ -2,13 +2,19 @@
 
 import requests
 
+from django.conf import settings
 from parsel import Selector
+
 from tests.support.dtos import CurrentDestinationStatus
 
 
-def current_topic_configuration(topic_name, host="localhost") -> CurrentDestinationStatus:
+def current_topic_configuration(
+    topic_name: str,
+    host: str = settings.STOMP_SERVER_HOST,
+    port: str = settings.STOMP_SERVER_INTERFACE_PORT,
```

Comment:
**suggestion:** Same thing here. Since these params have a default value, the type could be `Optional`.